lib/hobo_fields/model/field_spec.rb

```diff
@@ -20,12 +29,6 @@ def initialize(model, name, type, options={})
       TYPE_SYNONYMS = [[:timestamp, :datetime]]
 
       begin
-        MYSQL_COLUMN_CLASS = ActiveRecord::ConnectionAdapters::MysqlColumn
-      rescue NameError
-        MYSQL_COLUMN_CLASS = NilClass
-      end
-
-      begin
```

Comment:
This was a horrible idea of theirs!  This exactly class name was no longer correct with the mysql2 driver.